site/src/components/popover.md

```diff
@@ -50,58 +50,108 @@ You can apply almost any classes on the `summary` element to achieve the look yo
-</details>
+## Dynamic positioning
 
-<details class="popover popover-right margin-xxs">
-	<summary class="button">Popover aligned to the right</summary>
-	<div class="popover-content">Popover content.</div>
+By default, the popover content is centered with the popover button and opens downward, but will dynamically shift location to avoid overflowing outside of the window.
+
+```html
+<details class="popover margin-xxs">
+	<summary id="test-popover-summary" class="button">Popover</summary>
+	<div id="test-popover-content" class="popover-content">
+		Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+		labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+		laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+		voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+		non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+	</div>
 </details>
+
+<div class="display-flex justify-content-center">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+			voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+			non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+		</div>
+	</details>
+</div>
+
+<div class="display-flex justify-content-flex-end">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover</summary>
+		<div class="popover-content">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+			voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+			non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+		</div>
+	</details>
+</div>
 ```
 
-### Vertical alignment
+## Fixed positioning
 
-The popover can also show up above the button.
+You can also fix the position of the popover, which will disable dynamic positioning of the popover content.
 
 ```html
-<details class="popover popover-top margin-xxs">
-	<summary class="button">Popover on top</summary>
-	<div class="popover-content">Popover content.</div>
-</details>
+<div class="display-flex justify-content-center">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover top</summary>
+		<div class="popover-content popover-top">
```

Comment:
Is `popover-bottom` available? Please add an example or list the possible atomics so it's clear

Reply:
Oh, also, `popover-top` should be applied on `popover` level. Usually, we list all atomics on components level, so please revisit this as well

Reply:
added and updated